### DISCUSSION TITLE
commit prefix에 추가되면 좋을 것 각자 적어보면 좋을 것 같습니다!

### DISCUSSION THREAD

**Discussion by chadseok** [wanted-pre-onboarding-side-project/habit-tracker#46] [Ideas]
일단 댓글에다가 각자 추가되었으면 하는 prefix들 적어보고 적용은 의논 한 뒤에 하면 좋을 것 같습니다

**Comment by chadseok** [comment]
저는 일단 떠오르는 것은 3개 정도 있습니다! 😎

- move: 파일 이동 (이동한 이유도 커밋 메세지에 작성하기)
- remove: 파일 삭제 (삭제한 이유도 커밋 메세지에 작성하기)
- rename: 파일 이름 변경 (변경한 이유도 커밋 메세지에 작성하기)

**Reply by greyHairChooseLife** [reply]
디스커션 올려주신 덕에 한번 생각 해 봤습니다. 저는 당장은 뭔가 더 필요하다고 떠오르지 않아서 창환님이 제안 해 주신 부분에 질문만 좀 드려 보고 싶어요.

제안 해 주신 세 가지를 기존 refactor와 구분 지어서 얻을만한 것이 어떤 점들이라고 생각하시나요?

저도 remove가 따로 있으면 좋겠다는 생각을 했었어요. 다른 두 개는 이번에 보고서 refactor에 들어가는 거 아닌가 했고요. 근데 그러고 나서 보니까 그럼 remove도 refactor에 포함 되는 것 아닌가 하는 생각이 들었습니다. 

제가 기존에 remove가 refactor랑 별개로 구분된다고 생각 한 것은 CRA로 생긴 기본 파일들을 제거 할 때였거든요. 그 때는 코드를 개선한다는 것 보다는 그냥 필요 없는 것들을 지워버리는 것이니까요. 그 이후에 삭제 할 일이 언제 있는가를 따져 보면 결국 기존 코드를 개선 하는 refactor의 과정 하에 있는 것 같아요.
